plugins/api-docs/src/components/PlainApiDefinitionWidget/PlainApiDefinitionWidget.test.tsx

```diff
@@ -20,9 +20,10 @@ import { PlainApiDefinitionWidget } from './PlainApiDefinitionWidget';
 
 describe('<PlainApiDefinitionWidget />', () => {
   it('renders plain text', async () => {
-    const { getByText } = await renderInTestApp(
+    const { getByTestId } = await renderInTestApp(
       <PlainApiDefinitionWidget definition="Hello World" language="yaml" />,
     );
-    expect(getByText(/Hello World/i)).toBeInTheDocument();
```

Comment:
There is no `<textarea>Hello World</textarea>` anymore and `<CodeSnippet />` is rendering `Hello World` as following:

```html
      <span>
        <span
          style="color: rgb(33, 145, 97);"
        >
          Hello
        </span>
        <span>

        </span>
        <span
          style="color: rgb(33, 145, 97);"
        >
          World
        </span>
      </span>
```

Since now the text is split into multiple subcomponents, we though about using snapshot testing.
The downside of this approach is that we'll depend on `react-syntax-highlighter`: if the implementation changes, the test will also break.
Another approach could be to mock the implementation of `react-syntax-highlighter` and make it render the plain text instead.

Reply:
I'd say do the latter, speeds up the test as well. Snapshots are tricky to update later on imo, and often test too deep

Reply:
See also thus about textContent as an alternative https://github.com/testing-library/dom-testing-library/issues/201#issuecomment-484890360

Reply:
> See also this about textContent as an alternative [testing-library/dom-testing-library#201 (comment)](https://github.com/testing-library/dom-testing-library/issues/201#issuecomment-484890360)

love that 😍
Done.